CODE_OF_CONDUCT.md

```diff
@@ -1 +1,128 @@
-[//]: # (@Todo : Fill the file)
\ No newline at end of file
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
```

Comment:
Whaaaaat ? C'est quoi ce délire ? (après ya rien de particulier à faire, laisse le, ou vois avec Oliv !)

Reply:
Fais le voir a Oliv, je confirme. Tu lui indique que ca vient de code covenant etc etc.

Reply:
Typiquement c'est comme stripe c'est le truc de base de code of conduct